src/wp-includes/rest-api/endpoints/class-wp-rest-template-revisions-controller.php

```diff
@@ -164,14 +164,19 @@ public function register_routes() {
 	 * @return WP_Post|WP_Error Post object if ID is valid, WP_Error otherwise.
 	 */
 	protected function get_parent( $parent_post_id ) {
+		$error = new WP_Error(
+			'rest_post_invalid_parent',
+			__( 'Invalid template parent ID.' ),
+			array( 'status' => 404 )
+		);
+
+		if ( is_null( $parent_post_id ) ) {
```

Comment:
We usually use:

```suggestion
		if ( null === $parent_post_id ) {
```

What I don't like is that the `@param int $parent_post_id Supplied ID.` above says this will always be an `int`, when it clearly doesn't seem to be the case.

Reply:
I have changed the `is_null`.

Good catch about the `@param`! I have changed it to string as the `register_rest_route()` defines it as a string and the `get_block_template()` expects a string as input.